app/views/shelters/show.html.erb

```diff
@@ -9,13 +10,17 @@
     <p>Zip: <%= @shelter.zip %></p>
     <p><%= link_to("View this shelter's pets", "/shelters/#{@shelter.id}/pets", class: "name") %></p>
   </div>
-  <%= link_to 'New Review', "/shelters/#{@shelter.id}/new_review" %>
   <% @shelter.reviews.each do |review| %>
     <div class="card show" id="review-<%= review.id %>">
```

Comment:
style on these review cards is a little off too, need to modify/change this div's classes